client/index.js

```diff
@@ -1,19 +1,19 @@
-var load = require('load-script')
-var engineClient = require('engine.io-client')
+const load = require('load-script')
+const engineClient = require('engine.io-client')
```

Comment:
Please revert these, as well as the other changes in this file. We're using `var` to support old browsers.

If `standard` gives you trouble, you can use an [`eslint-disable <rule>` comment](https://eslint.org/docs/user-guide/configuring/rules#disabling-rules).